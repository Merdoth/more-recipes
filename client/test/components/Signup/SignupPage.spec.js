import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { SignupPage } from
  '../../../components/Signup/SignupPage.jsx';


const setup = () => shallow(<SignupPage />);

describe('SignupPage Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
