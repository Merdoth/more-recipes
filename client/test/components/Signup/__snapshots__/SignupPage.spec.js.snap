// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupPage Component snapshot it should render the right amount of elements 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-md-6 offset-md-3 cover"
    >
      <Connect(SignupForm)
        goToAllRecipes={[Function]}
        userSignupRequest={[Function]}
      />
    </div>
  </div>
</div>
`;
